configx/provider.go

```diff
@@ -476,26 +479,17 @@ func (p *Provider) CORS(prefix string, defaults cors.Options) (cors.Options, boo
 	}, p.Bool(prefix + "cors.enabled")
 }
 
-func (p *Provider) TracingConfig(serviceName string) *tracing.Config {
-	return &tracing.Config{
+func (p *Provider) TracingConfig(serviceName string) *otelx.Config {
+	return &otelx.Config{
 		ServiceName: p.StringF("tracing.service_name", serviceName),
 		Provider:    p.String("tracing.provider"),
-		Providers: &tracing.ProvidersConfig{
-			Jaeger: &tracing.JaegerConfig{
-				Sampling: &tracing.JaegerSampling{
-					Type:      p.StringF("tracing.providers.jaeger.sampling.type", "const"),
-					Value:     p.Float64F("tracing.providers.jaeger.sampling.value", float64(1)),
+		Providers: otelx.ProvidersConfig{
+			Jaeger: otelx.JaegerConfig{
+				Sampling: otelx.JaegerSampling{
 					ServerURL: p.String("tracing.providers.jaeger.sampling.server_url"),
 				},
-				LocalAgentAddress: p.String("tracing.providers.jaeger.local_agent_address"),
-				MaxTagValueLength: p.IntF("tracing.providers.jaeger.max_tag_value_length", jaeger.DefaultMaxTagValueLength),
-				Propagation: stringsx.Coalesce(
-					os.Getenv("JAEGER_PROPAGATION"),
-					p.String("tracing.providers.jaeger.propagation"),
-				),
-			},
-			Zipkin: &tracing.ZipkinConfig{
```

Comment:
Does this mean we don't support zipkin anymore?

Reply:
We do! It's all just simplified into one single configuration of a TextMapPropogator: https://github.com/ory/x/blob/dfef73e51c4a1de23d89b8696ae87545d82a1c39/otelx/jaeger.go#L81